pcdsdevices/lxe.py

```diff
@@ -315,8 +316,20 @@ def _setup_move(self, position):
         except Exception as ex:
             self.log.debug('Failed to update notepad setpoint to position %s',
                            position, exc_info=ex)
+
+        # Trigger dmov for tiny moves since done PV doesn't work here
+        if np.isclose(position, self.setpoint.get(), atol=20e-15, rtol=0):
```

Comment:
```suggestion
        short_move = np.isclose(position, self.setpoint.get(), atol=20e-15, rtol=0)
```

(Should be a multi-line suggestion; don't accept this one directly - even if you like it)

Reply:
ahh yes ok will do